NEWS.md

```diff
@@ -7,7 +7,8 @@ User-visible changes worth mentioning.
 
 ## master
 
-- [#] Add your PR description here.
+- [#1191]: Try to load bcrypt for application hashing, but add fallback.
```

Comment:
Maybe `secrets` instead of `application` ?

Reply:
Good point. I changed it to `Try to load bcrypt for hashing of application secrets, but add fallback.` to make sure its clear its related to hashing of the application secrets. Does that fit?

Reply:
Yep, it's ok, thanks